packages/firebase_core/lib/src/firebase_app.dart

```diff
@@ -6,44 +6,71 @@ part of firebase_core;
 
 class FirebaseApp {
   @visibleForTesting
-  const FirebaseApp({this.name, @required this.options});
+  const FirebaseApp({@required this.name}) : assert(name != null);
 
   /// Gets the name of this app.
```

Comment:
Should be `/// The name of this app.` per https://www.dartlang.org/guides/language/effective-dart/documentation#prefer-starting-variable-getter-or-setter-comments-with-noun-phrases